### PR TITLE
Adding Pokémon Breeder (Female)

### PR DESCRIPTION
**Pull request by TheOmniAdam**
Pokémon Breeder only has American English as a language. Skin uses outer
layer- not sure if that layer is enabled for npcs.

### CODE REVIEW

**Comment by MrMasochism** [comment]
The outer layer is enabled

**Comment by TheOmniAdam** [comment]
I seem to have created a conflict by re-ordering the NPC lines. I cleaned up the file and recommitted but that doesn't seem to change the fact that the branch is labeled with conflicts. Not sure what I need to do to resolve.

**Comment by isiliden** [comment]
Take your changes away, close the pull request
 Re-sync and re add your changes, then make a new pull request.

File: npcs/villagers/scientist_m_en_us.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
         {
             "lines": [
                 {
-                    "text": "The Pokédex says Cubone's mother died - but if EVERY Cubone's mother died..."
```
Comment (TheOmniAdam) [inline npcs/villagers/scientist_m_en_us.json]: Taking more correct version

File: npcs/villagers/scientist_m_en_us.json [hunk 2]
```diff
@@ -55,7 +55,7 @@
...
+                    "text": "The Pokédex says Cubone's Mother died- but if EVERY Cubone's Mother died..."
```
Comment (TheOmniAdam) [inline npcs/villagers/scientist_m_en_us.json]: Trying to resolve what appears to be a conflict?

**Pull request closed without merging**